.travis.yml

```diff
@@ -17,6 +17,17 @@ addons:
 
 matrix:
   include:
+    - node_js: lts/erbium
```

Comment:
nit: would be nice to put a comment here just to explain why we're doing this for our future selves